src/widget/wpushbutton.h

```diff
@@ -97,9 +108,12 @@ class WPushButton : public WWidget {
 
     // True, if the button is currently pressed
     bool m_bPressed;
+    // True, if the button is pointer is above button
+    bool m_bHovered;
 
     // Array of associated pixmaps
     int m_iNoStates;
+    Qt::TextElideMode m_elideMode;
```

Comment:
not initialized in the constructor.

Reply:
is it okay to do this in [setState()](https://github.com/mixxxdj/mixxx/pull/2471/files?file-filters%5B%5D=.cpp&file-filters%5B%5D=.h#diff-78d5371c502303528e220b3605c53534R251)?

Reply:
No, I guess code rule checkers will complain anyway. 
Add it just here: 
https://github.com/mixxxdj/mixxx/blob/be526c6ccfc30d78dfdd0cb65c80dffedb9e34e7/src/widget/wpushbutton.cpp#L39

Reply:
..and here https://github.com/mixxxdj/mixxx/blob/be526c6ccfc30d78dfdd0cb65c80dffedb9e34e7/src/widget/wpushbutton.cpp#L47
the other variables are initialized in setStates(). To reduce code duplication?

Reply:
Ups, yes setStates() is a good place for this. I was not aware that it is called from every constructor.